src/script/conversation/ConversationRepository.ts

```diff
@@ -184,8 +187,14 @@ export class ConversationRepository {
     // we register a client mismatch handler agains the message repository so that we can react to missing members
     // FIXME this should be temporary. In the near future we want the core to handle clients/mismatch/verification. So the webapp won't need this logic at all
     this.messageRepository.setClientMismatchHandler(async (mismatch, conversation, silent, consentType) => {
+      //we filter out self client id to omit it in mismatch check
+      const {userId, clientId} = this.core;
+      const selfClient = {domain: conversation?.domain, userId, clientId};
```

Comment:
Oh the domain need to be the user domain, not the conversation's one

Reply:
oops, fixed 👌